src/context.cpp

```diff
@@ -141,11 +149,24 @@ namespace librealsense
        assert(_device_watcher->is_stopped());
 
 #ifdef BUILD_WITH_DDS
-       _dds_watcher = std::make_shared< dds_device_watcher >( 0 );
```

Comment:
Do we really want a DDS watcher when we create a context from file? meaning for playback

Reply:
Hmm not sure... I can remove from here, I guess.

Reply:
I say let's remove it to save some CPU and potential bugs during playback..